src/webui/www/tstool.py

```diff
@@ -41,9 +41,10 @@
 www_folder = "."
 ts_folder = os.path.join(www_folder, "translations")
 
+
 def parseSource(filename, sources):
     print("Parsing %s..." % (os.path.normpath(filename)))
-    with open(filename, encoding = 'utf-8', mode = 'r') as file:
+    with open(filename, encoding='utf-8', mode='r') as file:
```

Comment:
Is the absence of spaces around `=` considered good formatting in Python? Although I'm not too surprised...

Reply:
AFAIK it is preferred to omit it for function parameters. Probably related to PEP 8. https://peps.python.org/pep-0008/